src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age !== undefined && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    <p>{`${person.sex}`}</p>
+    <p>{`${person.isMarried}`}</p>
+    <p className="Person__partner">
+      {`${
+        person.isMarried === false
+          ? 'I am not married'
+          : `${person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband`}`
```

Comment:
```suggestion
        !person.isMarried
          ? 'I am not married'
          : `${person.sex === 'm' ? `${person.partnerName} is my wife` : `${person.partnerName} is my husband`}`
```
pls move this expression to a separate const